embrace-test-common/src/main/kotlin/io/embrace/android/gradle/config/TestMatrix.kt

```diff
@@ -17,33 +17,34 @@ sealed class TestMatrix(
     val gradle: String,
     val kotlin: String,
     val jdk: JdkEnv,
+    val compileAndTargetSdk: String
 ) {
 
-    object UnsupportedOldGradleVersion : TestMatrix("7.4.2", "7.5", "1.9.25", JdkEnv.JAVA_11)
+    object UnsupportedOldGradleVersion : TestMatrix("8.0.0", "8.0", "1.9.25", JdkEnv.JAVA_17, "34")
 
     /**
      * The minimum version we support & run tests against.
      */
-    object MinVersion : TestMatrix("7.4.2", "7.5.1", "1.9.25", JdkEnv.JAVA_11)
+    object MinVersion : TestMatrix("8.0.2", "8.0.2", "1.9.25", JdkEnv.JAVA_17, "34")
 
     /**
      * Older than middle of the pack, but not as bad as our minimum.
      */
-    object OlderVersion : TestMatrix("8.1.4", "8.1.1", "1.9.25", JdkEnv.JAVA_17)
+    object OlderVersion : TestMatrix("8.1.4", "8.1.1", "1.9.25", JdkEnv.JAVA_17, "34")
 
     /**
      * Middle of the pack.
      */
-    object MiddleVersion : TestMatrix("8.3.2", "8.4", "1.9.22", JdkEnv.JAVA_17)
+    object MiddleVersion : TestMatrix("8.3.2", "8.4", "1.9.22", JdkEnv.JAVA_17, "35")
 
     /**
      * Not the latest, but newer than the middle of the pack.
      */
-    object NewerVersion : TestMatrix("8.5.2", "8.7", "2.0.0", JdkEnv.JAVA_17)
+    object NewerVersion : TestMatrix("8.5.2", "8.7", "2.0.0", JdkEnv.JAVA_17, "36")
 
     /**
      * The maximum version we currently run tests against. Newer versions may work, but are not
      * explicitly tested.
      */
-    object MaxVersion : TestMatrix("8.9.1", "8.13", "2.2.20", JdkEnv.JAVA_17)
+    object MaxVersion : TestMatrix("8.9.1", "8.13", "2.2.20", JdkEnv.JAVA_17, "36")
```

Comment:
It's probably worth tweaking the test matrix values in general so that there's a better spread of the versions we support - I'd like to see at least one version using Gradle 9, and the latest stable AGP version